products/models.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+
+class Category(models.Model):
+    name         = models.CharField(max_length=50)
+    created_at   = models.DateTimeField(auto_now_add=True)
+    
+    class Meta:
+        db_table = "categories"
+        
+class Subcategory(models.Model):
+    name         = models.CharField(max_length=50)
+    category     = models.ForeignKey("Category",on_delete=models.CASCADE, related_name='subcategories')
```

Comment:
snake_case로 수정 부탁드려요~!
```suggestion
    category     = models.ForeignKey("Category",on_delete=models.CASCADE, related_name='sub_categories')
```

Reply:
수정완료